recipes/deploy-admin.rb

```diff
@@ -116,7 +113,6 @@
         s3_distribution_bucket_name: s3_distribution_bucket_name,
         capture_agent_monitor_url: capture_agent_monitor_url,
         live_streaming_url: live_streaming_url,
-        using_ssl_for_engage: using_ssl_for_engage,
```

Comment:
Yes, I can see how this would be confusing. It is a fix to the engage integration that I missed when we switched to s3 distribution - I am passing the `using_ssl_for_engage` attribute to the template but not using it any more.  I caught it in this branch because of the work I was doing to restore a cluster's files across s3 buckets.